### PR TITLE
Update to uno 3 stable

### PR DESCRIPTION
**Pull request by GuillaumeSE**
﻿GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

avoid nuget uno ui conflict on external project that use a more recent stable version

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Other... Please describe:

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [x] Tested Android
- [ ] Tested WASM
- [ ] Tested MacOS
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,16 +95,7 @@ Here is what ColorPaletteOverride.xaml would contain if you want both light and
...
-			// Set a default palette to make sure all colors used by MaterialResources exist
-			this.Resources.MergedDictionaries.Add(new Material.MaterialColorPalette());
-			
-			// Overlap the default colors with the application's colors palette. 
-			// TODO: Replace ms-appx:///Views/ColorPaletteOverride.xaml with your resourceDictionary.
-			this.Resources.MergedDictionaries.Add(new ResourceDictionary() { Source = new Uri("ms-appx:///Views/ColorPaletteOverride.xaml") });
-			
-			// Add all the material resources. Those resources depend on the colors above, which is why this one must be added last.
-			this.Resources.MergedDictionaries.Add(new Material.MaterialResources());
-
```
Comment (agneszitte-nventive) [inline README.md:107]: We don't need to do that anymore with latest Uno ?

Comment (GuillaumeSE) [reply README.md:107]: like the lastest uno dev? or already available in the stable version?

Comment (MatFillion) [reply README.md:107]: I don't understand what updating Uno changes for this? it's still required to do it in 3 steps.

Comment (MatFillion) [reply README.md:107]: Ahh yes. Uno.Material.Resource.Init was added previously but it caused weird bugs so it was reverted.

Comment (GuillaumeSE) [reply README.md:107]: Ho! seems to work well for me until now with the latest uno stable. should I revert?

**Comment by MatFillion** [comment]
https://github.com/unoplatform/Uno.Material/pull/434
Will close this one, and use the above one instead

**Pull request closed without merging**